src/main/resources/application-prod.yaml

```diff
@@ -20,7 +20,7 @@ no.nav.security.jwt:
 
 spring:
   datasource:
-    url: jdbc:postgresql://A01DBVL011.adeo.no:5432/familie-oppdrag
+    url: jdbc:postgresql://A01DBVL035.preprod.local:5432/familie-oppdrag-15
```

Comment:
Denne peker på preprod, er det bestillt ny instans i prod?

Reply:
@hensol  johan har bestilt basen. Den er på A01DBVL035.adeo.no

Reply:
Virker som jeg kun når den basen fra utviklerimage

Reply:
For å nå den via naisdevice må man sette host til `prod-pg.intern.nav.no`

Reply:
Oppdaterte denne ettersom utviklerimage er "avviklet":
https://github.com/navikt/familie/blob/master/doc/utvikling/preprod/kikke_i_databasen.md

Fint om man legger inn riktig host for postgres15 også :)